pkg/cassandra/management.go

```diff
@@ -290,3 +295,20 @@ func (r *defaultManagementApiFacade) EnsureKeyspaceReplication(keyspaceName stri
 		}
 	}
 }
+
+func (r *defaultManagementApiFacade) GetSchemaVersions() (map[string][]string, error) {
+	pods, err := r.fetchDatacenterPods()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, pod := range pods {
+		if schemaVersions, err := r.nodeMgmtClient.CallSchemaVersionsEndpoint(&pod); err != nil {
+			r.logger.V(4).Error(err, "failed to list schema versions", "Pod", pod.Name)
```

Comment:
Why do we need `r.logger.V(4)` here?

Reply:
We don't; however, I think we should be setting verbosity most of the time. See #282.